package.json

```diff
@@ -89,7 +89,8 @@
     "start:benchmark": "ts-node ./src/start/startBenchmark.ts",
     "test": "npm run test:dev -- --coverage",
     "test:dev": "vitest run --passWithNoTests",
-    "test:types": "npm run lint:types"
+    "test:types": "npm run lint:types",
+    "prepare": "tsc"
```

Comment:
I am interested in the benefit of the `prepare` script? As far as I know it is automatically being executed when `npm publish` is invoked?

Reply:
It allows me to install this dependency from GitHub repository and not committing _dist/_ directory. It's being build during _npm install_

Reply:
Ok, let's remove it for now and bring it back in another PR as this is unrelated to the BB implementation. I would like to seperate these 2 concerns here.